### PR TITLE
amfora: new package at 1.8.0

### PR DESCRIPTION
**Pull request by Xeno-Hypster**
## Description of package
go client for gemini protocol

## Installed manifest of package

```
/var/db/kiss/installed/amfora/version
/var/db/kiss/installed/amfora/sources
/var/db/kiss/installed/amfora/manifest
/var/db/kiss/installed/amfora/depends
/var/db/kiss/installed/amfora/checksum
/var/db/kiss/installed/amfora/build
/var/db/kiss/installed/amfora/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/bin/amfora
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

### CODE REVIEW

File: community/amfora/build [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+make
+install -Dm755 -t "$1"/usr/bin amfora
```
Comment (git-bruh) [inline community/amfora/build:4]: style guide: quote the whole path instead of just `$1`

File: community/amfora/depends [hunk 2]
```diff
@@ -0,0 +1 @@
+go
```
Comment (git-bruh) [inline community/amfora/depends:1]: `go make`, its not a runtime dep

**Review by git-bruh — CHANGES_REQUESTED** [review]

File: community/amfora/depends (hunk 2 above)
Comment (Xeno-Hypster) [reply community/amfora/depends]: Fixed

File: community/amfora/build (hunk 1 above)
Comment (Xeno-Hypster) [reply community/amfora/build]: Fixed

**Pull request merged**